SensorNode/sensor_nodes/onBoxSensorNode.py

```diff
@@ -103,8 +126,18 @@ async def onRTPCommandResponse(self, packet: codec.binaryPacket):
             split_script = "-m ASM_utils.ffmpeg.split_log"
 
             cmd = (f'ffmpeg -f video4linux2 -input_format h264 -i {self.camera_endpoint}'
-                f' -vcodec copy -f mpegts tcp://{self.data_endpoint}:{endpoint_port} '
-                f' 2>&1 | {sys.executable} {split_script} {ff_stats_path} {ff_info_path}')
+                f' -vcodec copy -f mpegts tcp://{self.data_endpoint}:{endpoint_port} ')
+            
+            
+            
+            if self.extra_endpoints is not None:
+                for i in self.extra_endpoints:
```

Comment:
`i` should only be used for iterated indices - in this case, you are iterating over a list of strings.  Perhaps `for endpoint in self.extra_endpoints`?